src/taskcreator.js

```diff
@@ -79,7 +93,34 @@ class TaskCreator {
       this.lastCreateTask = {taskId: options.taskId, task};
       return {status: {taskId: options.taskId}};
     }
```

Comment:
Because of this `fakeCreate` conditional, it's hard to test the code that calls `appendLastFire`.  I think you were telling me that earlier, but I didn't understand :)

An alternative is to factor out the code that calls `queue.createTask` (basically the try/catch block below) into a separate method.  Then, in testing, just override that method on the `TaskCreator` instance with a dummy version.

Having done that, you can test both the "success" case (where the task-creation method returns something like `{result: 'success', taskId, time}`) and the "failure" case (`{result: 'error', taskId, error, time}`).